### ISSUE TITLE
(Bittrex) PT buys more than ALL_max_trading_pairs

### ISSUE THREAD

**Issue by krims0n32** [taniman/profit-trailer#470]
Had this happen three times now, with different PT versions (1.2.6.1 & 1.2.6.2). Seems it happens only on Bittrex, I have no issues on Binance.

max_trading_pairs is set to 5 for example and 5 pairs are in use, but PT will still buy one or two pairs more...

There is no error in the log when this happens.

**Comment by tcoffman** [comment]
I'm seeing the same issue on 1.2.6.2 and using bittrex as exchange. Issue just began in the last 3-4 hours.

**Comment by helo03** [comment]
This is a Bittrex Api-Call problem... So it's not related to PT, but to the exchange. We're searching for a way to fix this nevertheless.

**Comment by tcoffman** [comment]
So far the fix appears to be working. Can you elaborate on what changed with the API call? Which API call was it? And how is it being handled when the problem condition arises?

**Comment by razorseal** [comment]
I'm having the same problem... On bittrex, my setting is

ALL_max_trading_pairs = 5

However, I have 7 in DCA (which 2 were bought after I changed it from 8 to 5).

I turned on SOM to hope that it will clear out the DCAs and start fresh... Later same day, I turned SOM back off and it went ahead and purchased another one.

I upgraded to 1.2.6.6 today and hoping the issue is gone. This was with 1.2.6.1 I will comment back if it does it again.
